Readme.md

```diff
@@ -3,23 +3,50 @@
-<p><strong>See Also:</strong><br> <a href="https://www.devexpress.com/Support/Center/p/E2193">ASPxGridView - How to display popup using GridViewDataHyperLinkColumn and its NavigateUrlFormatString property</a></p>
+![Hyperlink Item Popup](hyperlink-popup.png)
 
-<br/>
+## Implementation Details
 
+In this example, the Popup Control's [ContentUrl](http://documentation.devexpress.com/#AspNet/DevExpressWebASPxPopupControlASPxPopupControl_ContentUrltopic) property value is calculated in the [ASPxHyperLink](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxHyperLink._members)'s [Init](https://docs.microsoft.com/en-us/dotnet/api/system.web.ui.control.init?view=netframework-4.8) event handler based on a data row field value and the base URL value stored in the session.
 
+The handler builds a JavaScript function that assigns the calculated `ContentUrl` value to the Popup Control on the client when a user clicks the link:
+
+```cs
+protected void hyperLink_Init(object sender, EventArgs e) {
+    ASPxHyperLink link = (ASPxHyperLink)sender;
+
+    GridViewDataItemTemplateContainer templateContainer = (GridViewDataItemTemplateContainer)link.NamingContainer;
+
+    int rowVisibleIndex = templateContainer.VisibleIndex;
+    string ean13 = templateContainer.Grid.GetRowValues(rowVisibleIndex, "EAN13").ToString();
+    string contentUrl = string.Format("{0}?EAN13={1}", Session["baseURL"], ean13);
+
+    link.NavigateUrl = "javascript:void(0);";
+    link.Text = string.Format("More Info: EAN13 - {0}", ean13);
+    link.ClientSideEvents.Click = string.Format("function(s, e) {{ OnMoreInfoClick('{0}'); }}", contentUrl);
+}
+```
+
+## Files to Look At
+
+- [Default.aspx](./CS/Solution/Default.aspx) (VB: [Default.aspx](./VB/Solution/Default.aspx))
+- [Default.aspx.cs](./CS/Solution/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Solution/Default.aspx.vb))
+- [Products.aspx](./CS/Solution/Products.aspx) (VB: [Products.aspx](./VB/Solution/Products.aspx))
+- [Products.aspx.cs](./CS/Solution/Products.aspx.cs) (VB: [Products.aspx.vb](./VB/Solution/Products.aspx.vb))
+
+## Documentation
+- [Popup Control](https://docs.devexpress.com/AspNet/3582/components/docking-and-popups/popup-control)
+
+
+## More Examples
+- [Popup Control for ASP.NET Web Forms - How to show a pop-up window](https://github.com/DevExpress-Examples/web-forms-show-popup-window)
+- [Popup Control for ASP.NET Web Forms - How to show and hide a pop-up window on the server side](https://github.com/DevExpress-Examples/asp-net-web-forms-popup-change-visibility)
+- [Grid View for ASP.NET Web Forms - How to display a hyperlink in templated column](https://github.com/DevExpress-Examples/aspxgridview-display-hyperlink-in-templated-column)
```

Comment:
Can you update this Readme.md title as follows?

`Grid View for ASP.NET Web Forms - How to display a hyperlink in a templated column`

Reply:
Thank you.